src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -21,12 +21,13 @@ export default class BraintreeHostedForm {
     private _cardNameField?: BraintreeRegularField;
     private _formOptions?: BraintreeFormOptions;
     private _type?: BraintreeHostedFormType;
+    private _isInitializedHostedForm?: boolean;
```

Comment:
Out of curiosity, why don't we initialise this as the variable as being `false`?

Reply:
Oh..Nice catch up. We can do that. Thanks 👍

Reply:
Fixed